### ISSUE TITLE
Missing game cover source *Broken?*

### ISSUE THREAD

**Issue by Kyzoku** [JosefNemec/Playnite#3501]
### Bug Description

After setting the Missing game cover source to Platform and adding an image in the Library Manager, the cover image does not show up.  The Icon and Background options work fine, just the cover does not.

### Diagnostics ID

cc2894f9-6595-48b9-84b2-7242e97d47cd

**Comment by JosefNemec** [comment]
Can't reproduce:
![image](https://github.com/JosefNemec/Playnite/assets/3874087/e72b06c5-653d-428c-8d61-643aa21fa1fb)

**Comment by JosefNemec** [comment]
No, so I guess that would be the issue.

**Comment by Kyzoku** [comment]
Yes, I just checked my grid view does show it, just not details view.
